pkgs/http/lib/src/utils.dart

```diff
@@ -74,3 +74,8 @@ Stream<T> onDone<T>(Stream<T> stream, void Function() onDone) =>
       sink.close();
       onDone();
     }));
+
+/// On upload progress callback.
+///
+/// See : "MultipartRequest"
+typedef OnUploadProgress = void Function(int uploaded, int total);
```

Comment:
I'd drop the typedef and just inline the function type.

No need to pollute the public, exported, API with an extra name.

Reply:
Done